pkgs/tools/admin/simp_le/default.nix

```diff
@@ -2,16 +2,16 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "simp_le-client";
-  version = "0.9.0";
+  version = "0.16.0";
 
   src = python3Packages.fetchPypi {
     inherit pname version;
-    sha256 = "1yxfznd78zkg2f657v520zj5w4dvq5n594d0kpm4lra8xnpg4zcv";
+    sha256 = "17azqlb1xsnh9p0m75apb19j7pramgj00cf5k6fwzz2zqz0x0hpp";
   };
 
   postPatch = ''
-    # drop upper bound of acme requirement
-    sed -ri "s/'(acme>=[^,]+),<[^']+'/'\1'/" setup.py
+    # drop acme bounds
```

Comment:
Should not we update (certbot) as https://github.com/NixOS/nixpkgs/pull/62283? Upstream must have bumped the lower bound for a reason.

Reply:
You're right. I did that in an attempt to quickly have something working. On it.

Reply:
Fixed, thanks for the heads up.